### ISSUE TITLE
False negative for jquery-ember-run when using saved jquery element reference 

### ISSUE THREAD

**Issue by tehhowch** [ember-cli/eslint-plugin-ember#1889]
```js
import $ from "jquery";
const $el = $("#item");
$el.on("click", () => this.handle());
```
passes, but it should fail.

I agree that the rule should not trigger for arbitrary `.on("click", callback)` scenarios, but when the property is known to be a jQuery element reference, it should trigger.

**Comment by bmish** [comment]
Yeah that would be a good case to catch. Would require using our `getNodeOrNodeFromVariable()` helper (which uses `findVariable` from `eslint-utils` under the hood) to detect it.
